app/src/main/java/org/wikipedia/dataclient/mwapi/MwServiceError.kt

```diff
@@ -72,4 +76,11 @@ class MwServiceError(val code: String? = null,
                 return expiry.after(now)
             }
     }
+
+    @Serializable
+    open class AbuseFilter {
```

Comment:
Do we actually need to include/deserialize this structure? We're not using any of these fields, and it looks like your addition of prioritizing "`abusefilter-`" codes is all that's necessary here.

If we deserialize things without using them, we risk hidden errors if the structure given by the server isn't exactly what we expect.

Reply:
Thanks! Forgot to remove the class after testing. Done!